### DISCUSSION TITLE
Problem regarding the submission via Tortoise GIT

### DISCUSSION THREAD

**Discussion by david54573** [DLR-VSDC/IEEE-MVC-2023#37] [Q&A] [answered]
When we tried to push our simulation model to GitHub, some error messages appeared, resulting in a failure. 

The error information mainly showed 66 unknown Git LFS objects. 

Can you help me solve the problem? 

Can I complete the push by initializing Git? However, this means that the original version management information from you will disappear.

![Screenshot 2023-02-23 152730](https://user-images.githubusercontent.com/119647012/220843337-e2ef1a03-3f9b-422d-848c-fecec50af0fc.png)

**Comment by imeb** [comment]
@tobolar Could you please help with this issue? Thanks in advance.

**Reply by tobolar** [reply]
Done.

**Comment by tobolar** [comment, marked as answer]
(Optional, just to get info all LFS-files are captured, run:
```
git lfs ls-files
```
Files like mat, fmu, pdf, etc. shall be listed.)

---

Please open CLI (see below how to start it) and try to run 
```
git lfs fetch --all
```
first. Git  shall report something like 
```
fetch: 44 objects found, done.
fetch: Fetching all references...
[f88879ffbeae7ed9107246c3b55ff864b78b4a569623aacabd1909a0734575f1] Object does not exist on the server: [404] Object does not exist on the server
[bff5abcb23e0802d90c577853403f88ecff737aa7e890502111af29982cda986] Object does not exist on the server: [404] Object does not exist on the server
...
```
After that, you can push again. 
```
git push -u origin main
```

---

### Start CLI

Btw. to start git CLI, you can right mouse click anywhere in your repository and select *Git Bash Here*:

![grafik](https://user-images.githubusercontent.com/9588978/221527961-c43296fc-dc6a-4bda-90ae-182b60af428b.png)


---

### Push using Tortoise

Commiting all files and pushing using *TortoiseGit 2.12.3.0* runs smoothly at my side - so I unfortunately cannot reproduce the error:

![grafik](https://user-images.githubusercontent.com/9588978/221523392-e95d9612-fc1d-436e-a41d-78c41b90b981.png)
